components/footer/index.tsx

```diff
@@ -25,7 +25,7 @@ export default function Footer() {
           alt="The Algorithms logo"
         />
         <Typography className={classes.name}>
-          &#169; The Algorithms 2021
+          &#169; The Algorithms 2022
```

Comment:
We could update the year automatically by using the `Date` constructor in TypeScript.
You'd need to add this code after line 8, then you'll have to use that variable in the text.

```typescript
const currentYear = new Date().getFullYear();
```

```suggestion
          &#169; The Algorithms {currentYear}
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient